pkg/runner/runner.go

```diff
@@ -918,7 +937,7 @@ func (runner *Runner) GetAirMemorySegmentsAddresses() (map[string]AirMemorySegme
 	segmentsOffsets, _ := runner.vm.Memory.RelocationOffsets()
 	memorySegmentsAddresses := make(map[string]AirMemorySegmentEntry)
 	for segmentIndex, segment := range runner.vm.Memory.Segments {
-		if segment.BuiltinRunner == nil {
+		if segment.BuiltinRunner.String() == "no builtin" {
```

Comment:
Why this change?

Reply:
because the default runner is &NoBuiltin, with `.String()` method returning "no builtin". Isn't it quicker to compare strings, rather then interfaces?

Reply:
You are correct, for some reason I had thought it was a pointer